app/src/main/java/com/codesoom/assignment/domain/Task.java

```diff
@@ -0,0 +1,87 @@
+package com.codesoom.assignment.domain;
+
+import com.codesoom.assignment.common.Builder;
+
+import java.time.LocalDateTime;
+import java.util.UUID;
+
+public class Task {
+    private String taskId;
+    private String title;
+    private LocalDateTime createdAt;
+    private LocalDateTime updatedAt;
+    private Status status;
+
+    private enum Status{
+        REMOVE,INCOMPLETE,COMPLETE
+    }
+    private Task(TaskBuilder taskBuilder) {
+        this.title = taskBuilder.title;
+        this.taskId = taskBuilder.taskId;
+    }
+
+    public Task registerTask() {
+        this.status=Status.INCOMPLETE;
+        this.createdAt = LocalDateTime.now();
+        this.updatedAt = LocalDateTime.now();
+        this.taskId = UUID.randomUUID().toString()
+                .replaceAll("-", "")
+                .substring(0, 8);
+        return this;
+    }
+
+    /*
+         task아이디를 통헤서 기존에 등록된 task를 가져온 후
+         title과 update시간을 변경할 예정입니다.
+     */
```

Comment:
이렇게 할 일을 주석으로 남기는 경우에는 `TODO` 태그를 입력해 주세요. 찾기도 쉽고 IDE도 관리를 도와줄 것입니다.

```suggestion
    /*
         TODO: task아이디를 통헤서 기존에 등록된 task를 가져온 후
         title과 update시간을 변경할 예정입니다.
     */
```